docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -22,10 +22,10 @@ NOTE: KQL autocomplete for `.alerts-security.alerts-*` indices is available on t
 `Detection alerts` is selected.
 
 * Use the date and time filter to define a specific time range. By default, this filter is set to search the last 24 hours.
-* View detection alerts generated by a specific rule. To do this, go to *Detect* -> *Rules*, then select a rule name in the All rules table. The *Rules detail page* displays a comprehensive view of the rule's details, and alert details are displayed in the Alerts table beneath the Trend histogram. Alternatively, you can go to *Detect* -> *Alerts* -> *Manage rules*, then click on a rule name in the All rules table to view the alerts associated with the rule.
+* View detection alerts generated by a specific rule. Go to *Detect* -> *Rules*, then select a rule name in the All rules table. The rule details page displays a comprehensive view of the rule's settings, and the Alerts table under the Trend histogram displays the alerts associated with the rule (including alerts from any previous or deleted revision of that rule).
```

Comment:
```suggestion
* View detection alerts generated by a specific rule. Go to *Detect* -> *Rules*, then select a rule name in the All rules table. The rule details page displays a comprehensive view of the rule's settings, and the Alerts table beneath the Trend histogram displays the alerts associated with the rule, including alerts from any previous or deleted revision of that rule.
```

Reply:
@jmikell821 Changing the parentheses to a comma looks good to me, but any particular reason to revert "under" back to "beneath"? I originally changed that because "beneath" feels slightly archaic, and "under" is simpler.